pkg/http/middlewares.go

```diff
@@ -25,28 +25,39 @@ func NewZapMiddleware(logger *zap.Logger) negroni.Handler {
 
 		next(writer, req)
 
+		negroniWriter, hasStatus := writer.(negroni.ResponseWriter)
+		if hasStatus {
+			logger = logger.With(
+				zap.Int("status", negroniWriter.Status()),
+				zap.Int("body_bytes_sent", negroniWriter.Size()),
+			)
+		}
+
 		logger = logger.With(
-			zap.Int("status", writer.(negroni.ResponseWriter).Status()),
-			zap.Int("body_bytes_sent", writer.(negroni.ResponseWriter).Size()),
 			zap.Float32("duration", float32(time.Since(start))/float32(time.Second)),
 		)
 
-		switch {
-		case writer.(negroni.ResponseWriter).Status() >= 500:
+		if !hasStatus {
+			logger.Info(req.URL.String())
+			return
+		}
+
+		switch negroniWriter.Status() {
+		case http.StatusInternalServerError:
 			logger.Error(req.URL.String())
-		case writer.(negroni.ResponseWriter).Status() >= 400:
+		case http.StatusBadRequest:
 			logger.Warn(req.URL.String())
 		default:
 			logger.Info(req.URL.String())
 		}
```

Comment:
The behavior change here; a `504` will log as `Info`. Golang allows `switch` without key for these cases. It's probably better to disable the linter here because it is what we want.

Reply:
Oops I missed that, I thought it was an ==
Thanks